### ISSUE TITLE
Add support for Assembla?

### ISSUE THREAD

**Issue by jlubeck** [bitrise-io/bitrise-webhooks#31]
I wouldn't know how to write something like this in order to make a pull request. But is there something I can do to help? 

This is how the webhooks setup looks like on Assembla:
![Alt text](https://monosnap.com/file/E5rZCnO652w9pwXrRoAGpAfjKl8CW1.png)

**Comment by viktorbenei** [comment]
> I wouldn't know how to write something like this in order to make a pull request.

just follow the guide in the README: https://github.com/bitrise-io/bitrise-webhooks#how-to-compile--run-the-server & https://github.com/bitrise-io/bitrise-webhooks#development

**Comment by viktorbenei** [comment]
also https://github.com/bitrise-io/bitrise-webhooks#how-to-add-support-for-a-new-provider - just start at https://github.com/bitrise-io/bitrise-webhooks#how-to-compile--run-the-server and follow the README ;)

**Comment by viktorbenei** [comment]
In short (copy paste from the README's relevant section):

> Implement your webhook provider support code into ./service/hook/theprovider. Unit tests are required if you want your code to be merged into the main bitrise-wekhooks repository!

> Once the implementation is ready add it to the selectProvider function, to the supportedProviders list, in the service/hook/endpoint.go file.

> For an example you should check the service/hook/github (single webhook triggers only one build) or the service/hook/bitbucketv2 (a single webhook might trigger multiple builds) provider implementation.

And of course we're happy to help as much as we can ;)

**Comment by viktorbenei** [comment]
Migrated this feature request to: https://discuss.bitrise.io/t/assembla-webhook-support/222
